_docs/_latest/use/model.md

```diff
@@ -59,5 +59,7 @@ $ forseti model delete <MODEL_NAME>
 
 The command above will delete the data model with model name = `<MODEL_NAME>`.
 
+## What's next
+
 Once we have the data model ready, you can learn how to query the data model using Forseti Explainer
 [here]({% link _docs/latest/use/explain.md %}).
```

Comment:
Learn how to query the data model using [Explain] ...

Reply:
done

Reply:
it's not Explainer, it's Explain.